interface/src/app/signature/participation-type-selector.tpl.html

```diff
@@ -0,0 +1,7 @@
+<div class="alert alert-danger" role="alert" ng-if="$ctrl.errorMessage">
+  {{$ctrl.errorMessage}}
+</div>
+<select name="selectedParticipationType"
```

Comment:
Ok, I think I will put the description in the signature page template itself rather than here so that how the description is displayed is not hard-coded into this component.